### ISSUE TITLE
The big problem with the way keys are handled. 

### ISSUE THREAD

**Issue by PseudonymPatel** [Michael-Reeves-Discord-Community/Michael-Reeves-Community-Bot#10]
Its CEO of Swift here with another github issue...
Generally, the you have a separate file containing solely the keys, and the file will be imported into whatever needs it. The file, of course, is put in the gitignore. Some types of files that are commonly used is environment variables, a .env file (python-dotenv does this and environment vars), yml or json files. This allows different people to use their own personal keys, not have to ever worry about leaking keys, not having to update a tracked file with a fake key... 

As we've seen, the keys are always at risk of getting leaked, and there are definitely more professional, better ways of doing things.

**tl;dr: Use a secrets file!!**

**Comment by shrig1** [comment]
ye, I will talk to cloud about this

**Comment by fccloud42** [comment]
Yep, we were thinking about doing something like this with github secrets, but I don't think any of us know powershell. If we could decide on a file type then I think that this is a way that we could deal with it. I will look more into the dotenv library, seems pretty good from the skimming I've done so far.

**Comment by shrig1** [comment]
Okay so turns out the Github secrets are used if you have the Github actions

**Comment by PseudonymPatel** [comment]
there is no need to know powershell if you use .env which are handled easily with one line of code (using python-dotenv)

**Comment by PseudonymPatel** [comment]
fixed
